socorro/tests/external/es/test_crashstorage.py

```diff
@@ -204,144 +213,91 @@ def test_index_crash_mapping_keys(self):
             "uuid": old_uuid,
         }
 
-        es_storage.save_processed_crash(
+        crashstorage.save_processed_crash(
             raw_crash=raw_crash,
             processed_crash=processed_crash,
         )
 
-        self.es_context.refresh()
+        es_helper.refresh()
 
         # Retrieve the document from this week and verify it has the user_comments
         # field
-        doc = self.conn.get(
-            index=self.es_context.get_index_for_date(now),
-            id=now_uuid,
-        )
-        assert field in doc["_source"]["processed_crash"]
+        with es_helper.conn() as conn:
+            doc = conn.get(
+                index=crashstorage.get_index_for_date(now),
+                id=now_uuid,
+            )
+            assert field in doc["_source"]["processed_crash"]
 
         # Retrieve the document from four weeks ago and verify it doesn't have the
         # user_comments field
-        doc = self.conn.get(
-            index=self.es_context.get_index_for_date(four_weeks_ago),
-            id=old_uuid,
-        )
-        assert field not in doc["_source"]["processed_crash"]
+        with es_helper.conn() as conn:
+            doc = conn.get(
+                index=crashstorage.get_index_for_date(four_weeks_ago),
+                id=old_uuid,
+            )
+            assert field not in doc["_source"]["processed_crash"]
 
 
-class TestESCrashStorage(ElasticsearchTestCase):
+class TestESCrashStorage:
     """These tests are self-contained and use Mock where necessary"""
 
-    def setup_method(self):
-        super().setup_method()
-        self.config = self.get_tuned_config(ESCrashStorage)
-
-    def test_index_crash(self):
-        """Mock test the entire crash submission mechanism"""
-        es_storage = ESCrashStorage(config=self.config)
-
-        # This is the function that would actually connect to ES; by mocking
-        # it entirely we are ensuring that ES doesn't actually get touched.
-        es_storage._submit_crash_to_elasticsearch = mock.Mock()
-
-        es_storage.save_processed_crash(
-            raw_crash=deepcopy(SAMPLE_RAW_CRASH),
-            processed_crash=deepcopy(SAMPLE_PROCESSED_CRASH),
+    def build_crashstorage(self):
+        return build_instance(
+            class_path="socorro.external.es.crashstorage.ESCrashStorage",
+            kwargs=settings.CRASH_DESTINATIONS["es"]["options"],
         )
 
-        # Ensure that the indexing function is only called once.
-        assert es_storage._submit_crash_to_elasticsearch.call_count == 1
-
-    @mock.patch("socorro.external.es.connection_context.elasticsearch")
-    def test_success(self, espy_mock):
+    def test_indexing_success(self, es_helper):
         """Test a successful index of a crash report"""
```

Comment:
I think this has already been tested. We should merge this with the integration test above.